### PR TITLE
DOCS-24 - RBAC for Indexes - GA

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request removes beta tagging from the following article, creates a release note, and make other updates for the GA: 
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes/

## Ticket 

[DOCS-24](https://sumologic.atlassian.net/browse/DOCS-24)

(The following Asana ticket was deleted because it was migrated to Jira: "[RBAC for Indexes -  GA])(https://app.asana.com/0/1183741177809740/1205147588069408/f)"

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@manashar4, could you review this pull request?

File: blog-service/2024-02-02-manage.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Role Based Search Index (Manage)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - rbac
+  - index
+  - roles
+  - release notes
```
Comment (kimsauce) [inline blog-service/2024-02-02-manage.md:8]:
```suggestion
```

**Review by kimsauce — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]
nicely written!

**Comment by jpipkin1** [comment]
Thank you, @kimsauce. 😄

**Review by kimsauce — APPROVED** [review]

**Pull request merged**